### PR TITLE
Add Digitools to Learning and Courses / Groupware

### PR DESCRIPTION
**Pull request by kokomo123**
I think Digitools might be fitting in the Misc category / groupware as from the rough translation from French. I also tried digitools.app, wasn't a valid website (lead to a weird blog) Linked from #295 

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/digitools.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "Digitools"
+website_url: "https://ladigitale.dev/digitools"
+source_code_url: "https://codeberg.org/ladigitale/digitools"
+description: "A set of simple tools to accompany the animation of courses in person or from afar. (documentation in French)."
```
Comment (Ki-er) [inline software/digitools.yml:4]: From afar reads a bit wierd. Remotely is a more standard term. 

```suggestion
description: "A set of simple tools to accompany the animation of courses in person or remotely. (documentation in French)."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick tweak.

File: software/digitools.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
+tags:
+  - Groupware
+  - Miscellaneous
```
Comment (Ki-er) [inline software/digitools.yml:11]: > animation of courses
```suggestion
tags:
  - Learning and Courses
  - Groupware
```

File: software/digitools.yml (hunk 1 above)
Comment (kokomo123) [reply software/digitools.yml:4]: Remotely was a weird term for me to figure out, but I'll roll with it 👍🏻

**Review by nodiscc — APPROVED** [review]
Thanks

**Pull request merged**